hack/jenkins/e2e-runner.sh

```diff
@@ -179,35 +190,35 @@ function dump_cluster_logs() {
     fi
 }
 
-# GCI specific settings.
-# Assumes: JENKINS_GCI_IMAGE_FAMILY
-function setup_gci_vars() {
-  local -r gci_staging_project=container-vm-image-staging
-  local -r image_info="$(gcloud compute images describe-from-family ${JENKINS_GCI_IMAGE_FAMILY} --project=${gci_staging_project} --format=json)"
-  local -r image_description="$(echo ${image_info} | jq -r '.description')"
-  local -r image_name="$(echo ${image_info} | jq -r '.name')"
-
-  if [[ "${JENKINS_USE_GCI_VERSION:-}" =~ ^[yY]$ ]]; then
-    # GCI QA jobs use the builtin k8s version.
+# Figures out the builtin k8s version of a GCI image.
+function get_gci_k8s_version() {
+    local -r image_description=$(gcloud compute images describe ${KUBE_GCE_MASTER_IMAGE} --project=${KUBE_GCE_MASTER_PROJECT})
```

Comment:
Unfortunately the image description is now just a string and not programatically formatted. I do have a change pending to make this version number detection more reliable though. Just wanted to get the jobs started soon.